### PR TITLE
Robert carleton prep

### PR DESCRIPTION
**Pull request by robertlucyshyn**
Add minimum area criterion to filter out phantom targets due to noise (as seen in drive practice at Woodroffe High School on March 8, 2020).

John: Please add a similar change to MergeFRCPipeline.py as well.

### CODE REVIEW

File: FindTarget.py [hunk 1]
```diff
@@ -438,19 +438,23 @@ def findTape(contours, image, centerX, centerY, mask, CornerMethod, MergeVisionP
...
 
     if len(contours) >= 1:
         # Sort contours by area size (biggest to smallest)
         cntsSorted = sorted(contours, key=lambda x: cv2.contourArea(x), reverse=True)[:10]
        
         cntsFiltered = []
 
-        if cntsSorted:
+        # First contour has largest area, so only go further if that one meets minimum area criterion
+        if cntsSorted and cv2.contourArea(cntsSorted[0]) > minContourArea:
```
Comment (jogray72) [inline FindTarget.py]: I don't understand why you would check only the first value here for Area?    Why not just check when the rest of the filters are checked?

Also, we  have a method called checkTargetSize() that used area.  It has been removed.   Why not just have a method called checkTargetFingerPrint() or something and do all the fingerprinting in one method?

Comment (jogray72) [reply FindTarget.py]: Another point is you are doing another call to cv2.contourArea() when it should only have to be called once.

Comment (robertlucyshyn) [reply FindTarget.py]: Updated based on our conversation. Please have a look at our changes

**Comment by jogray72** [comment]
Looks good.  Thanks Robert!

**Pull request merged**